CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ Breaking changes:
 New features:
 
 Bugfixes:
+- Updated dependencies to clear build errors related to unlisted dependencies (#24)
```

Comment:
Would you mind moving this to the “Other improvements” section, as it’s a purely internal change? You can also add your username if you would like — ie (#24 by @flounders)

Reply:
No problem, I just updated it.